clinica/pipelines/statistics_volume/statistics_volume_cli.py

```diff
@@ -0,0 +1,112 @@
+__credits__ = ["Arnaud Marcoux"]
+__license__ = "See LICENSE.txt file"
+__version__ = "0.3.0"
+__maintainer__ = "Arnaud Marcoux"
+__email__ = "arnaud.marcoux@icm-institute.org"
+__status__ = "Development"
+
+import clinica.engine as ce
+
+
+class StatisticsVolumeCLI(ce.CmdParser):
+
+    def define_name(self):
+        """Define the sub-command name to run this pipeline."""
+        self._name = 'statistics-volume'
+
+    def define_description(self):
+        """Define a description of this pipeline."""
+        self._description = ('Brief description:\n'
+                             'http://clinica.run/doc/Pipelines/Statistics_Volume/')
+
+    def define_options(self):
+        """Define the sub-command arguments."""
+        from clinica.engine.cmdparser import PIPELINE_CATEGORIES
+
+        clinica_comp = self._args.add_argument_group(PIPELINE_CATEGORIES['CLINICA_COMPULSORY'])
+
+        clinica_comp.add_argument("caps_directory",
+                                  help='Path to the CAPS directory.')
+
+        clinica_comp.add_argument("subject_visits_with_covariates_tsv",
+                                  help='TSV file containing a list of subjects with their sessions and all '
+                                       'the covariates and factors needed for the GLM.')
+
+        clinica_comp.add_argument("contrast",
+                                  help='Defines the contrast. Must be one of the column names form the TSV file.')
+
+        clinica_comp.add_argument("feature_type",
+                                  help='Define what type of file are grabbed for the analysis. Use "fdg" or "graymatter". Use your own if you want to use the --custom_file flag. ')
+
+        clinica_comp.add_argument("group_id",
+                                  help='User-defined identifier for the provided group of subjects.')
+
+        optional = self._args.add_argument_group(PIPELINE_CATEGORIES['OPTIONAL'])
+
+        optional.add_argument("--custom_files", "-cf", type=str, default=None,
+                              help=('Custom file string. Specify filename using * when the subject or session name appear. '
+                                    + 'Example : \'*_task-rest_acq-fdg_pet_space-Ixi549Space_pet.nii.gz\' will grab the'
+                                    + ' corresponding file in all the subjects/sessions'))
+
+        optional.add_argument("-tup", "--threshold_uncorrected_pvalue",
+                              type=float, default=0.001,
+                              help='Threshold to display the uncorrected p-value '
+                                   + '(--threshold_uncorrected_pvalue 0.001).')
+
+        optional.add_argument("-tcp", "--threshold_corrected_pvalue",
+                              type=float, default=0.05,
+                              help='Threshold to display the corrected p-value '
+                                   '(default: --threshold_corrected_pvalue 0.05)')
```

Comment:
I think these flags are either not used or have wrong help message. Is this a fast copy/paste from `statistics-surface`?

Reply:
I do not really know what else to say about them (and yes it is a copy paste)

Reply:
Ok. Simply remove these lines, there is no equivalent in your code.